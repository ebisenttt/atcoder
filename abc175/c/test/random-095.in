-350758243755215 566736195173730 595909076701523
