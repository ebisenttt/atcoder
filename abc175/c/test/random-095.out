350758243755215
